### DISCUSSION TITLE
new UI sucks

### DISCUSSION THREAD

**Discussion by devendranaga** [community/community#66498] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

I understand that your engineers / product managers want to work on features. But we all know "Do not fix what's not broken" and "Do not change UI experience so drastically that it completely unusable".

What is required in the feed:

1. What the projects i am working on / following have checkins from contributors. I am interested in these.
2. Bugs and PRs.

Everything else section / feed:
1. Interesting projects.
2. Feeds that may be suitable based on my search history / project visits.

What do we have:

1. unholy mess that looks like a next version of twitter / facebook or text version of instagram

As a dev i do not want to see this. This will not help me increase productivity nor will help me focus on my work. Heck, this does not even let me understand what i want to follow unless i stare 10 mins at it.

It makes me wonder, are the teams developing this at GitHub not see what they are doing? Are you not using your own product?

I do not want a social media page.

Thanks

**Comment by ShahbazShaddy** [comment]


**Reply by fdwr** [reply]
🤔 Was this reply automatically generated via ChatGPT? It reads rather robotic/automated.

**Comment by SamJakob** [comment]
I have been gathering methods (browser extension and userscript) to bring the old one back here:
https://github.com/orgs/community/discussions/66249

Also a poll/discussion on crowdsourcing one that actually is more useful.

**Comment by ettaboyle** [comment]
_We've added the following update to the [Updates to your GitHub Feed](https://github.com/orgs/community/discussions/65343#top) Discussion Announcement originally posted August 29, 2003._


## _Sept. 12th Update_
_With the latest feed update on September 6, 2023, we made changes to the underlying technology of the feed to improve overall platform performance. As a result, we removed the functionality for “push events for repositories a user is subscribed to”. We don’t take these changes lightly, but as our community continues to grow tremendously, we have to prioritize our availability, user experience, and performance._ 


_Thanks to feedback from the community, we have updated our [changelog](https://github.blog/changelog/2023-09-06-updates-to-your-github-feed/) to clarify and have fixed the following bugs from the initial September 6th release:_


- _We were showing releases for organizations that you follow, instead of only showing them for repositories that you watch or star. This is now fixed._ 
- _We have also addressed a problem where 'Followed You' cards were sometimes appearing out of chronological order._
- _We do not plan to re-include “push events for repositories a user is subscribed to” for the reasons listed above, but we will continue to address feedback as it aligns with our platform goals._


_We understand that many of you are upset with the recent changes to your feed. We should have done a better job communicating recent changes and how those decisions relate to our broader platform goals. Your continued feedback is invaluable as we evolve and continue to strive to provide a first-class developer experience that helps every developer be happier and more productive._

**Comment by SamJakob** [comment]
> _Announcement originally posted August 29, 2003._

That’s some rather impressive planning there! 🤔

—

In my personal opinion I don’t have an issue with the removal of push notifications - releases are a useful tool for maintainers to indicate a significant set of changes (i.e., relevant push notifications) as long as they’re shown to the right people (i.e., those that actually subscribed to them and expect to see them for a given repository - so not starred, just watching) so showing all pushing is just noise.

First push is kind of covered by repository creation.

Maybe it would be possible to aggregate and cache what a user one is following has pushed to? e.g., “X made recent changes to Y and Z” for the feed as a suitable alternative to the other cases.

—

Aside from that, my biggest concerns with the new feed are quite literally that every item takes up far too much room for what it describes and I would enjoy more granular control over what I see (more than just selecting topics allow) - e.g., I want to see releases from repositories I’ve subscribed to, but not just every repository I’ve starred however different people use stars (amongst other things) and, indeed, the platform differently and hence would want to filter out certain events within a group differently.

Additionally, I would like the option to filter out specific repositories or people if they’re too “noisy” as it makes things difficult to keep track of (e.g., if someone is starring loads of repositories, I’d like for the number of items I see about that to be reduced or condensed).

—

(Also it’s a must that the feed be chronological!)
